FinalWithLED/main.asm

```diff
@@ -0,0 +1,110 @@
+;
+; FinalWithLED.asm
+;
+; Created: 11/28/2018 10:25:47 AM
+; Author : Mizia
+;
+
+
+start:
+	ldi r29, 0x00
+	out tcnt0, r29		;clear initial value of clk
+	;set the RGB LED
+	sbi ddra, 1
+	sbi ddra, 2
+	sbi ddra, 3
+    sbi porta, 1
+	sbi porta, 2
+	sbi porta, 3
+	
+	;set sensor and buzzer
+	cbi ddrb, 0		;input sensor
+	sbi ddrb, 1		;ouput sensor
+	sbi ddra, 0		;input buzzer
+	 
+loop1:

```

Comment:
This (and many other labels) does not offer a lot of help in understanding your code. Try to come up with more meaningful names.